### PR TITLE
Edited grammar and presentation

### PR DESCRIPTION
**Pull request by thedowntownerd**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ Our full-stack web development curriculum is completely free and self-paced. We
...
+Each of these 30 projects in the freeCodeCamp.org curriculum have their own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.
 
 You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on websites like CodePen and Glitch - or even on your local computer's development environment.
 
-Once you’ve earned a certification, you will always have it. You will always be able to link to it from your LinkedIn or résumé. And when your prospective employers or freelance clients click that link, they’ll see a verified certification specific to you.
+Once you’ve earned a certification, you will always have it. You will always be able to link to it from your LinkedIn or Résumé. And when your prospective employers or freelance clients click that link, they’ll see a verified certification specific to you.
```
Comment (RandellDawson) [inline README.md]: Do not capitalize resume.  Also, it should not have accent marks.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
 
-Some of the contribution workflows, like previewing pages for the guide or the coding challenges, debugging and fixing bugs in codebase, requires you to have freeCodeCamp running locally.
+Some of the contribution workflows, like previewing pages for the guide or the coding challenges, debugging and fixing bugs in codebase, require you to have freeCodeCamp running locally.
 
 ## Fork the repository on GitHub
 ['Forking'](https://help.github.com/articles/about-forks/) is a step where you get your own copy of freeCodeCamp's main repository (a.k.a _repo_) on GitHub.
 
-This is essential as it allows you to work on your own copy of freeCodeCamp on GitHub, or to download (clone) your repository to work on locally. Later, you will be able to request changes to be pulled into the main repository from your fork via a pull request.
+This is essential as it allows you to work on your copy of freeCodeCamp on GitHub, or to download (clone) your repository to work on locally. Later, you will be able to request changes to be pulled into the main repository from your fork via a pull request.
```
Comment (RandellDawson) [inline docs/how-to-setup-freecodecamp-locally.md]: Please revert this line:
```suggestion
This is essential as it allows you to work on your own copy of freeCodeCamp on GitHub, or to download (clone) your repository to work on locally. Later, you will be able to request changes to be pulled into the main repository from your fork via a pull request.
```

File: README.md [hunk 3]
```diff
@@ -26,11 +26,11 @@ Our full-stack web development curriculum is completely free and self-paced. We
 
 freeCodeCamp.org offers several free developer certifications. Each of these certifications involves building 5 required web app projects, along with hundreds of optional coding challenges to help you prepare for those projects. We estimate that each certification will take a beginning programmer around 300 hours to earn.
 
-Each of these 30 projects in the freeCodeCamp.org curriculum has its own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.
+Each of these 30 projects in the freeCodeCamp.org curriculum have their own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.
```
Comment (RandellDawson) [inline README.md]: Please revert.
```suggestion
Each of these 30 projects in the freeCodeCamp.org curriculum has its own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.
```

**Comment by raisedadead** [comment]
Hi @thedowntownerd since you have made the pull requests from `master` its now in a soiled state and cant be merged as is. Please seek help in our [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors)

**Pull request closed without merging**